bool draw_coords  # draw/hide coordinates
bool draw_segments  # draw/hide segmentation outcome
---
bool success  # bool feedback
